pkg/cli/debug/debug.go

```diff
@@ -204,7 +205,14 @@ func NewCmdDebug(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra
 		Run: func(cmd *cobra.Command, args []string) {
 			kcmdutil.CheckErr(o.Complete(cmd, f, args))
 			kcmdutil.CheckErr(o.Validate())
-			kcmdutil.CheckErr(o.RunDebug())
+
+			cmdErr := o.RunDebug()
+			if o.IsNode {
```

Comment:
this really only applies to debugging node targets? @soltysh ?

Reply:
nevermind, this is clarified in https://github.com/openshift/oc/pull/1123#issuecomment-1139520896. however we should add a small godoc to say why.